src/main/java/mimeparser/MimeMessageConverter.java

```diff
@@ -298,6 +297,18 @@ public String replace(Matcher m) throws Exception {
                     ADD_HEADER_IFRAME_JS_TAG_TEMPLATE, tmpHtmlHeader.toURI(), Resources.toString(contentScriptResource, StandardCharsets.UTF_8));
         }
 
+        // Append attachment filename list to body
+        if (addAttachmentNames) {
+            String attachmentsHtml = "";
+            List<AttachmentResource> attachments = EmailConverter.mimeMessageToEmail(message).getAttachments();
```

Comment:
I wonder what the performance penalty for this is, but since it's behind a flag, it's ok.